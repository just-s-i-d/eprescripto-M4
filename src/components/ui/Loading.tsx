import capsulePng from "@assets/capsule.png";

const Loading = () => {
  return (
    <div className="flex justify-center items-center h-screen">
      <img
        src={capsulePng}
        className="z-1 animate-loading"
        width={70}
        alt="loader"
      />
    </div>
  );
};
export default Loading;
