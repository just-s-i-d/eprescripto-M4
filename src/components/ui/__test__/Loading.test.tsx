import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Loading from "../Loading";

describe("Loading test", () => {
  it("Title test", () => {
    render(<Loading />);
    expect(screen.getByAltText("loader")).toBeInTheDocument();
  });
});
