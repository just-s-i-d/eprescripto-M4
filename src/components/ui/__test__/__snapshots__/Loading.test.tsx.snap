// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading test Snapshot test 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center items-center h-screen"
  >
    <img
      alt="loader"
      class="z-1 animate-loading"
      src="capsule.png"
      width="70"
    />
  </div>
</DocumentFragment>
`;
